vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/loader/LocalJavaProjectLoader.java

```diff
@@ -66,7 +65,7 @@ public Optional<IJavaProject> load(String uri, Map<Event, Consumer<String>> hand
     {
         return RuntimeWrapper.wrapMono(() ->
         {
-            File projectFolder = Paths.get(new URI(uri).getPath()).toFile();
+            File projectFolder = new File(new URI(uri).getPath());
```

Comment:
which issue is solved here?

Reply:
The uri string has form \d:\path1\path2\test-project, new File(..) normalizes the form